node/blockchain/views/open_api_examples/account_state.py

```diff
@@ -0,0 +1,15 @@
+from drf_spectacular.utils import OpenApiExample
+
+account_state_open_api_example = OpenApiExample(
```

Comment:
Let's define it with models which get serialized and also reuse it with fixtures, so a change in the code automatically reflected in the documentation, otherwise the docs quickly become outdated